ex03.ex

```diff
@@ -74,14 +30,11 @@ defmodule Ex03 do
 
       iex> Ex03.list_contains([ 1, 2, 3, 4], 3)
       true
-
   """
 
-  def list_contains . .. "your code"
-
-  ##############################################################################
-  # 3.3:  5 points #
-  ##################
+  def list_contains([x | _t], x), do: true
+  def list_contains([_1 | t], x), do: list_contains(t, x)
+  def list_contains(_, _x), do: false
```

Comment:
Again this is too permissive, as it allow me to write

~~~ elixir
list_contains(:wombat, 1)
~~~

I'd suggest the last function head should be

~~~ elixir
def list_contains([], _x), do: false
~~~

I'm not docking any marks—the point was already made in 3.1